src/constants/featureFlags.ts

```diff
@@ -1,3 +1,4 @@
 export const FEATURE_FLAGS = {
```

Comment:
can we add deploy notes to the PR mentioning to turn on this flag for prod during release

Reply:
done